src/View/Helper/FormHelper.php

```diff
@@ -1008,7 +1008,8 @@ public function input($fieldName, array $options = [])
             'required' => null,
             'options' => null,
             'templates' => [],
-            'templateVars' => []
+            'templateVars' => [],
+            'optionsLabel' => true
```

Comment:
`optionsLabel` is even ambigious in a way. It could take an array of key/values where the key is the option value and the value is the option label (for radio/checkboxes) or <option>-tag conntent (for select/multi select).